README.md

```diff
@@ -210,6 +210,39 @@ Command: <code>python3 -m pip install --user --upgrade pip</code>
 	Link to Solution: <a href="https://github.com/Udayraj123/OMRChecker/issues/70#issuecomment-1268094136">#70</a>
 </details>
 
+### Running in a Container Runtime (Docker or Podman)
+
+The repo contains a containerfile that can be used to run ORM checker without 
+interfering with a system python installation.
+The following instructions are based on using `docker` but can be adapted to
+`podman` by exchanging the names of the two programs.
+
+
+The container can be built by:
+
+1. Cloning the repository.
+```bash
+git clone https://github.com/Udayraj123/OMRChecker
+cd OMRChecker/
+```
+1. Building the container image.
+```bash
+docker build -f containerfile -t ormchecker:python3.11-slim
```

Comment:
Getting an error for this command on macos-
```
ERROR: "docker buildx build" requires exactly 1 argument.
See 'docker buildx build --help'.

Usage:  docker buildx build [OPTIONS] PATH | URL | -

Start a build
```

I think the correct command is to add a '.' at the end, can you confirm the same?

Reply:
Let's also add the output of `docker image ls` to confirm expected output:
```
REPOSITORY                          TAG               IMAGE ID       CREATED          SIZE
omrchecker                          python3.11-slim   3823e16d89c9   33 seconds ago   1.74GB
```